bootstrap/postgres/configs.go

```diff
@@ -483,7 +493,7 @@ func (cr configRepository) retrieveAll(owner string, filter bootstrap.Filter) (s
 	return fmt.Sprintf(template, f), params
 }
 
-func (cr configRepository) rollback(content string, tx *sqlx.Tx) {
+func (cr configRepository) rollback(_ string, tx *sqlx.Tx) {
```

Comment:
Most of cr.rollback call in this file sents the content. But it is not wrapped here. 
Could you please wrap the content with error?